soccer/gameplay/plays/restarts/placement.py

```diff
@@ -59,8 +59,8 @@ def on_enter_placing(self):
                                  required=True,
                                  priority=90)
 
-        #line_up = tactics.line_up.LineUp(self.create_lineup())
-        #self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
+        line_up = tactics.line_up.LineUp(self.create_lineup())
+        self.add_subbehavior(line_up, 'line_up', required=True, priority=80)
```

Comment:
We could just make a large line shaped obstacle from the ball to the placement position. This would cause our robots to move out of the way of any reasonable method of placing the ball while still getting as close to where we want them as they could.